src/context/game-context.tsx

```diff
@@ -0,0 +1,22 @@
+import React, { createContext, Reducer, useContext, useReducer } from "react";
+import { dataReducer } from "./Reducer/dataReducer";
+import { ChildrenProps, ContextType, Question } from "./game-context.types";
+
+const GameContext = createContext<ContextType | null>(null);
+
+export const GameProvider = ({ children }: ChildrenProps) => {
+  const [dataState, dataDispatch] = useReducer(dataReducer, {
+    questions: [],
+  });
+
+  return (
+    <GameContext.Provider value={{ dataState, dataDispatch }}>
+      {children}
+    </GameContext.Provider>
+  );
+};
+
+export const useGameContext = () => {
+  const data = useContext(GameContext);
+  return data!;
```

Comment:
after applying the above changes here you can directly write
`return useContext(GameContext)`

Reply:
I did this because,I have the initial state as null,So to assert that it wont be null later I used it.